README.md

```
@@ -96,11 +104,150 @@ pnpm run start:server -- --webhook-url 1234567890:http://localhost:4000/webhook
+```javascript
+const crypto = require('crypto');
+
+function verifyWebhookSignature(signature, secret, body) {
+  const expectedSignature = 'sha256=' +
+    crypto.createHmac('sha256', secret).update(body).digest('hex');
+  return signature === expectedSignature;
+}
+```
+
+**Example:**
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)